packages/orangutan/webpack.config.js

```diff
@@ -121,20 +121,17 @@ const config = {
   entry: webpackEntry,
   output: {
     path: path.resolve(__dirname, 'dist/'),
-    filename: '[name].[hash].bundle.js',
+    filename: '[name].[chunkhash].bundle.js',
     chunkFilename: '[name].[chunkhash].chunk.js',
-    library: 'orangutan',
+    library: pkgJson.name,
     libraryTarget: 'umd',
   },
   optimization: {
     minimize: true,
     splitChunks: {
-      maxInitialRequests: 6,
-      maxAsyncRequests: 6,
+      chunks: 'initial',
       minChunks: 1,
       cacheGroups: {
-        default: false,
-        vendors: false,
```

Comment:
🙋‍♂️Are we going to use the [default cache groups](https://gist.github.com/sokra/1522d586b8e5c0f5072d7565c2bee693#configurate-cache-groups) ? 
I thought the `cacheGroups` would be manually configured.

Reply:
The document doesn't explain how webpack handles `default` and `vendors` cacheGroups if we provide custom cacheGroups.
My test result is that even though we remove `default: false` and `vendors: false`, 
webpack still won't build `default` and `vendors` chunks for us.

I can add them back if we have concerns.

Reply:
I guess it's because the number of forced sub-chunks is already over the default `maxInitialRequests`, so even there's no `default: false`,  webpack still won't build default and vendors chunks for us

Reply:
So... if i understand correctly, `default` and `vendors` might have a chance to be built if modules which do not belong to other `cacheGroups` meets some default configuration (such as [this gist](https://gist.github.com/sokra/1522d586b8e5c0f5072d7565c2bee693) says).

Is it correct ?

Reply:
Yes, @YuCJ 's guessing is right.
The chunk number is over `splitChunks.maxInitialRequests: 3`,
so `defaults` and `vendors` cacheGroups won't be bundled.

Reply:
> So... if i understand correctly, `default` and `vendors` might have a chance to be built if modules which do not belong to other `cacheGroups` meets some default configuration (such as [this gist](https://gist.github.com/sokra/1522d586b8e5c0f5072d7565c2bee693) says).
> 
> Is it correct ?

Yes, it is correct.
However, `default` and `vendors` cacheGroups are nice to have.
`default` will extract repeated codes (minChunks: 2) into another bundle.
`vendors` will bundle the remaining files under `node_modules`

Reply:
In the future, if `default` and `vendors` are built,  it indicates that we have a chance to optimize the webpack bundles again. Thus, I would like to keep them here.

@taylrj  how do you think?

Reply:
I second that.
After a second thought, it is better to keep them here.